src/aleph_client/commands/program.py

```diff
@@ -83,8 +84,36 @@ def upload(
         help="Hash of the runtime to use for your program. Defaults to aleph debian with Python3.8 and node. You can also create your own runtime and pin it",
     ),
     beta: bool = typer.Option(False),
+
+
+
     debug: bool = False,
     persistent: bool = False,
+    persistent_volumes: Optional[Tuple[str, str, str]] = typer.Option(
+        (None, None, None),
+        help= '''Takes 3 parameters                                                                  
+        A persistent volume is allocated on the host machine at any time
+        eg:host my-volume 1 '''),
+
+    ephemeral_volumes: Optional[List[str]] = typer.Option(
+        None,
+        help=
+        '''Takes 1 parameters 
+        Ephemeral volumes can move and be removed by the host,
+        Garbage collected basically, when the VM isn't running  eg: 100 '''),
+
+    immutable_volumes: Optional[Tuple[str, bool,str]] = typer.Option(
```

Comment:
```suggestion
    immutable_volume: Optional[List[Tuple[str, bool,str]]] = typer.Option(
```

Reply:
Ah, and please format before committing 
```suggestion
    immutable_volumes: Optional[Tuple[str, bool, str]] = typer.Option(
```